index.js

```diff
@@ -1 +1,5 @@
-// Code your solution here
+const findMatching = (drivers, name) => drivers.filter(driver => driver.toLowerCase() === name.toLowerCase())
+
+const fuzzyMatch = (drivers, name) => drivers.filter(driver => driver.substring(0, 1) === name.substring(0, 1))
```

Comment:
So this works, and passes the test, but not for all examples (if the name provided was "Ste", this would provide a false positive for "Stove" since it's only looking at the first letter of either). Take a peak at the `startsWith` function. You don't need to make changes, but if you have extra time, play around with it. Thanks!

Reply:
Yeah, the fuzzy match was weird. The tests were asking that that it does not return drivers if the middle or ending letters match. But wanted it to pass for the beginning of the name. What is the beginning of a name? The first letter, two letters, three letters, etc.

Reply:
And I will go back and refactor this using the startsWith function and see if I find it easier to understand using that function.

Reply:
Use console logs and read the tests and you'll see what I mean. Thanks TJ.